lib/cocina/models/request_file.rb

```diff
@@ -2,23 +2,11 @@
 
 module Cocina
   module Models
-    # A request to create a file.
+    # Metadata for a request to create a File.
```

Comment:
This literally is the request itself, not just the metadata.

Reply:
I will fix these comments, thanks.